extension/core_functions/scalar/date/current.cpp

```diff
@@ -29,7 +29,9 @@ static void CurrentDateFunction(DataChunk &input, ExpressionState &state, Vector
 static void CurrentTimestampFunction(DataChunk &input, ExpressionState &state, Vector &result) {
 	D_ASSERT(input.ColumnCount() == 0);
 
-	auto val = Value::TIMESTAMPTZ(GetTransactionTimestamp(state));
+	auto ts = GetTransactionTimestamp(state);
+	auto ts_tz = Cast::Operation<timestamp_t, timestamp_tz_t>(ts);
```

Comment:
The casting here doesn't really do anything - `timestamp_t` and `timestamp_tz_t` are the same (they both hold micros since epoch).

Reply:
Yes, I added it more for completeness - but I can remove this cast entirely and directly reassign the value of `timestamp_t` to `timestamp_tz_t`.